common/app/views/fragments/chapterHeadings.scala.html

```diff
@@ -4,10 +4,20 @@
 
 @if(ChapterHeadingsSwitch.isSwitchedOn && content.chapterHeadings.size > 2) {
     <div class="chapters">
-        <span class="chapters__heading">Contents</span>
-        <ol>
+        <h3 class="chapters__heading">Chapter guide</h3>
+        <!-- THIS NEEDS TO BE FINISHED -->
+        <button class="chapters__button js-toggle-chapters meta-button hide-on-desktop" data-link-name="weather-toggle-chapters" href="#toggle-chapters">
```

Comment:
This arrow is still going to be visible even on core when the JavaScript isn't running. You can use the "modern-visible" class to make it only show on modern browsers (not in core).